pkg/api/v1/types.go

```diff
@@ -993,14 +993,16 @@ type ReplicationControllerSpec struct {
 	// Template is the object that describes the pod that will be created if
 	// insufficient replicas are detected. This takes precedence over a
 	// TemplateRef.
-	Template *PodTemplateSpec `json:"template,omitempty" description:"object that describes the pod that will be created if insufficient replicas are detected; takes precendence over templateRef"`
+	Template       *PodTemplateSpec `json:"template,omitempty" description:"object that describes the pod that will be created if insufficient replicas are detected; takes precendence over templateRef"`
+	SequenceNumber int              `json:"sequenceNumber" description:""`
 }
 
 // ReplicationControllerStatus represents the current status of a replication
 // controller.
 type ReplicationControllerStatus struct {
 	// Replicas is the number of actual replicas.
-	Replicas int `json:"replicas" description:"most recently oberved number of replicas"`
+	Replicas       int `json:"replicas" description:"most recently oberved number of replicas"`
+	SequenceNumber int `json:"sequenceNumber" description:""`
```

Comment:
I figured people will change what i write anyway, so I'd just leave it blank till someone says: write this :)